plugins/inputs/sflow/types/packets.go

```diff
@@ -0,0 +1,285 @@
+package types
```

Comment:
Is there a reason "types" and "packetdecoder" are packages?  Now that we've switched to a straightforward coding style, there's not enough complexity to need separate dirs and packages.

It seems like packetdecoder/* should be moved to the main dir and types/packets.go should be moved to types.go in the main dir.  The name types.go is clearer to me than packets.go.

Reply:
valid point. It's mostly for flexibility in how the types are used with other decoders/encoders, but I don't really think we have that scenario here, so tighter coupling would be fine.